qiskit/version.py

```diff
@@ -85,22 +85,22 @@ def _get_qiskit_versions():
     out_dict = {}
     out_dict['qiskit-terra'] = __version__
     try:
-        from qiskit.providers import aer
+        import qiskit_aer as aer
         out_dict['qiskit-aer'] = aer.__version__
     except Exception:
         out_dict['qiskit-aer'] = None
     try:
-        from qiskit import ignis
+        import qiskit_ignis as ignis
         out_dict['qiskit-ignis'] = ignis.__version__
     except Exception:
         out_dict['qiskit-ignis'] = None
     try:
-        from qiskit.providers import ibmq
+        import qiskit_ibmq_provider as ibmq
         out_dict['qiskit-ibmq-provider'] = ibmq.__version__
     except Exception:
         out_dict['qiskit-ibmq-provider'] = None
     try:
-        from qiskit import aqua
+        import qiskit_aqua as aqua
```

Comment:
They're mostly for clarity it's not strictly required because the old namespaces will continue to work (and will continue in the future, otherwise this approach wouldn't have been tenable). I can change it back if you think that would be better